GOVERNANCE.md

```diff
@@ -26,7 +26,7 @@ CTC on an ongoing basis.
 Individuals making significant and valuable contributions are made
 Collaborators and given commit-access to the project. These
 individuals are identified by the CTC and their addition as
-Collaborators is discussed during the weekly CTC meeting.
+Collaborators are discussed during the weekly CTC meeting.
```

Comment:
I disagree with this change. A better change might be to make 'as Collaborators' to instead read 'to the Collaborators'.